ga4gh/frontend.py

```diff
@@ -384,8 +403,139 @@ def searchDatasets(version):
         version, flask.request, app.backend.searchDatasets)
 
 
+# The below paths have not yet been implemented
+
+
+@app.route('/<version>/callsets/<no(search):id>')
```

Comment:
If there are two routes, 
```
/<version>/callsets/search
/<version>/callsets/<id>
```
a request to `/someVersion/callsets/search` will be routed to the later, which is not what we want.  Therefore, we have this converter to explicitly say that the `<id>` attribute of the later should not match on a literal `search`, thus routing the request correctly.